modules/backing-up-red-hat-quay.adoc

```diff
@@ -0,0 +1,251 @@
++
+[source,terminal]
+----
+$ oc -n <quay-namespace> rsh $(oc get pod -l app=quay -o NAME -n <quay-namespace> |head -n 1) cat /conf/stack/config.yaml|awk -F"/" '/^DB_URI/ {print $4}'
+quayregistry-quay-database
+----
+
+. Download a backup database:
++
+[source,terminal]
+----
+$ oc exec quayregistry-quay-database-59f54bb7-58xs7 -- /usr/bin/pg_dump -C quayregistry-quay-database  > backup.sql
+----
+
+. Create and download a backup database:
++
+----
+$ oc exec example-registry-quay-database-66969cd859-n2ssm -- /usr/bin/pg_dump -C example-registry-quay-database > backup.sql
+----
++
+The backup of your Quay database will be saved in the `backup.sql` file. If you are using an external database, refer to Migrating a standalone Quay deployment to a {productname} Operator managed deployment.
+
+. Decode and export the `AWS_ACCESS_KEY_ID`:
++
+[source,terminal]
+----
+$ export AWS_ACCESS_KEY_ID=$(oc get secret -l app=noobaa -n quay-enterprise  -o jsonpath='{.items[0].data.AWS_ACCESS_KEY_ID}' |base64 -d)
+----
+
+. Decode and export the `AWS_SECRET_ACCESS_KEY_ID`:
++
+[source,terminal]
+----
+$ export AWS_SECRET_ACCESS_KEY=$(oc get secret -l app=noobaa -n quay-enterprise -o jsonpath='{.items[0].data.AWS_SECRET_ACCESS_KEY}' |base64 -d)
+----
+
+. Create a new directory and copy all blobs to it:
++
+[source,terminal]
+----
+$ mkdir blobs
+----
++
+[source,terminal]
++
+----
+$ aws s3 sync --no-verify-ssl --endpoint https://$(oc get route s3 -n openshift-storage  -o jsonpath='{.spec.host}')  s3://$(oc get cm -l app=noobaa -n quay-enterprise -o jsonpath='{.items[0].data.BUCKET_NAME}') ./blobs
+----
+
+
+[NOTE]
+====
+You can also use link:https://rclone.org/[rclone] or link:https://s3tools.org/s3cmd[sc3md] instead of the AWS command line utility.
+====
+
+. Set the `horizontalpodautoscaler` back to a `managed` state in your QuayRegistry custom resource and remove the `replicas` override for the Quay component:
++
+[source,terminal]
+----
+$ oc patch QuayRegistry <quay-registry-name> -n <quay-registry-namespace>  -p '{"spec": {"components": [{"kind": "horizontalpodautoscaler", "managed": true }]}}' --type='merge'
+----
++
+Example YAML:
++
+[source,yaml]
+----
+  apiVersion: quay.redhat.com/v1
+  kind: QuayRegistry
+  metadata:
+    name: registry
+    namespace: ns
+  spec:
+    components:
+      …
+      - kind: horizontalpodautoscaler
+        managed: true
+      - kind: quay
+        managed: true
+      …
+----
++
+[NOTE]
+====
+If you are using a version older than 3.7.0, then you must manually scale down the Operator and manually scale down the Quay deployment. For example, you can run `$ oc scale --replicas=1 deployment quay-operator.v3.6.2 -n openshift-operators` to scale down the Operator, and `$ oc scale --replicas=1 deployment QUAY_MAIN_APP_DEPLOYMENT` to scale down the main application deployment.
+====
+
+. Check the status of the Operator:
++
+[source,terminal]
+----
+$ oc get quayregistry <registry-name> -n <quay-namespace> -o yaml
+----
```

Comment:
For consistency, do we need to replace `<registry-name>` by `<quay-registry-name>`?